sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/ExternalCatalogSuite.scala

```diff
@@ -164,15 +164,6 @@ abstract class ExternalCatalogSuite extends SparkFunSuite with BeforeAndAfterEac
     assert(actual.tableType === CatalogTableType.EXTERNAL)
   }
 
-  test("create table when the table already exists") {
```

Comment:
since in `InMemoryCatalog` and `HiveExternalCatalog` does not check if the table exists, here move the test case to `SessionCatalogSuite` and `HiveExternalCatalogSuite`